app/views/measures/_measure.html.erb

```diff
@@ -51,11 +51,15 @@
   <% end %>
 
   <td class="conditions-col govuk-table__cell">
-    <% if measure.has_measure_conditions? %>
-      <%= link_to "Conditions",  "#", class: 'reference', 'data-popup-ref' => "#{measure.destination}-#{measure.id}-conditions", "aria-label": "Click to open a dialog with the conditions for this measure", role: "button" %>
-    <% elsif show_proofs_for_geographical_areas?(roo_schemes, measure) %>
-      <%= link_to "Conditions",  "#", class: 'reference', 'data-popup-ref' => "#{measure.destination}-#{measure.id}-cds-proofs", "aria-label": "Click to open a dialog with the conditions for this measure", role: "button" %>
-    <% end %>
+    <div data-controller="anchor">
+      <% if measure.has_measure_conditions? %>
+        <%= link_to "Conditions", "#", class: 'reference', data: { action: "click->anchor#launchModal", modal_ref: "#{measure.destination}-#{measure.id}-conditions" }, "aria-label": "Click to open a dialog with the conditions for this measure", role: "button" %>
+        <div id="modal" class="modal tariff-info" data-controller="modal" data-anchor-target="modal"></div>
```

Comment:
Do we need one of these within each render of `_measure` when there's only a possibility of one modal at once?  If we can move this into some higher level spot we'll only need the one with the same Id.

Reply:
Modal Div must sit within an Anchor Div.  Attempted to move Anchor Div to higher level than row cell but Anchor Controller was out of scope for modal controller to launch modal.  So have moved one of the Modal Divs out of the if statement.